nss-cohort-36/bangazon-api-imagination-station-api
# Ts customer view

# Description

Customer view created in API. Should be able to create a customer and login as an existing customer via Postman.

## Type of change
Please delete options that are not relevant.
# Testing Instructions

1. `git fetch --all`
1. `git checkout TS-customer-view`
1. Open Postman with http://127.0.0.1:8000/register/, select POST, and use this in your body to create a user: `{
	"username": "JamCat",
	"password": "666killall",
	"first_name": "Jam",
	"last_name": "Cat",
	"email": "jc@email.com"
}`
1. Check Tableplus to make sure new Customer is present in django_auth and bangazonapi_customers.
1. Open Postman with http://127.0.0.1:8000/login/, select POST, and use this in your body to login a user: `{
	"username": "JamCat",
        "password": "666killall"
}`
1. You should receive a token on registering and logging in.

# Checklist:
